python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
We should not add anything to the `extraOpts` attribute. The mixture of the Ganga internals and users adding to the same attributes is causing trouble. In particular it is ill defined when a job is copied. Adding a flag that controls if this is added seems fine to me.